packages/client-node/src/index.ts

```diff
@@ -1,6 +1,7 @@
+export type { NodeClickHouseClient } from './client'
```

Comment:
Maybe it's better to re-export as just `ClickHouseClient`; WDYT @mshustov?

Reply:
You export it at the bottom, don't you? if you ask about a type - yes, we can re-export it with a more generic name to shorten the name